mongodbatlas/containers.go

```diff
@@ -27,7 +27,7 @@ type ContainersServiceOp service
 
 var _ ContainersService = &ContainersServiceOp{}
 
-type ContainersListOptions struct {
```

Comment:
Let's try to minimize the impact of this change by not renaming the struct, if a client is using this then we're are going to break their usage